fix-point/app/Main.hs

```diff
@@ -1,4 +1,15 @@
 module Main where
 
+import System.Directory (doesDirectoryExist)
+import System.Environment (getArgs)
+
+import CopyFiles (copyAll)
+import Rename
+
 main :: IO ()
-main = putStr "someFunc"
+main = do
+    args <- getArgs
```

Comment:
`getArgs` works well but it's very basic. Now you need to implement true command-line arguments parser using `optparse-applicative` library. The interface should be the following:

1. `fix-point somename` — creates project with given name `somename` and `Lib` module and directory are named as `Somename`.
2. `fix-point somename --prefix OtherName` — creates project with given name `somename` and `Lib` prefix is replaced with `OtherName`.
3. `fix-point --source-path some/path/to/three-layer` — takes path to `three-layer` repository.

Of course, it should be possible to use specify multiple options like

```
fix-point foo --prefix Bar --source-path some/path/to/source
```